src/Microsoft.AspNetCore.Razor.Language/Extensions/DefaultTagHelperTargetExtension.cs

```diff
@@ -99,7 +100,7 @@ public void WriteTagHelperBody(CodeRenderingContext context, DefaultTagHelperBod
                 var uniqueId = (string)context.Items[CodeRenderingContext.SuppressUniqueIds];
```

Comment:
I left this as-is. The ability to suppress might still be useful in some cases like unit tests.